.pylintrc

```diff
@@ -173,9 +173,18 @@ missing-member-max-choices=1
 
 [VARIABLES]
 
-# List of additional names supposed to be defined in builtins. Remember that
-# you should avoid to define new builtins when possible.
+# List of additional names supposed to be defined in builtins.
+# Remember that you should avoid to define new builtins when possible.
```

Comment:
I only just noticed this. Under very few circumstances should the `.pylintrc` file be altered. This is not one of those circumstances. Please use `pylint: disable=pylint-failure` within the code if you feel that the errors are not applicable, or try to resolve them if possible. Altering this file makes it difficult for us to update it using a script if we make global changes.

Please revert all of these changes and add `disable`s to your code to handle any issues.

Reply:
As a side note - I would prefer the `disable`s and potential `enable`s included not be global. This means I would prefer them to be in-line. However, if you find you're using the same disable repeatedly, it's ok to use a global `disable` but please explain why  if you do. Thanks!

Reply:
ok will do this..

Reply:
done.